### ISSUE TITLE
Add the participation table visible for all participants

### ISSUE THREAD

**Issue by giovanisp** [fairdemocracy/vilfredo-client#328]
The participation table is currently visible only for the question author, and only in writing phase.
The table should appear to all participants of a question.
The table should appear during writing phase (without the column "evaluated proposals")
The table should appear during the voting phase (with both writing and evaluated columns)

A new ticket will be opened about the table layout that can be optimized for more data (@giovanisp)

![image](https://cloud.githubusercontent.com/assets/180794/8144044/197c8d60-11ca-11e5-9752-55df163c77f0.jpg)

**Comment by derekpa** [comment]
I disagree that the participation table should be available to all participants. It gives them too much information, about who authored the proposals for instance, which reduces fairness. 

@pietrosperoni? What does your mathematical brain say, specifically about fairness?

**Comment by pietrosperoni** [comment]
Hmm, it looks like there are several type of questions, and several type of communities which we need to serve. And they have different needs. 

Indeed knowing who wrote proposals (and who didn't) can make people change their voting behaviour. But according to this also the questioner (the person who asks the question) should not see it.

Which is why we want to have blind moderation.

Also this might change when we introduce anonymising names to make people anonymous. 

I would say, let's wait until we have anonymity on and external blind moderation on. And then this issue might simply evaporate.

**Comment by derekpa** [comment]
What do you mean, wait. Make the table visible to voters who have voted or not?

As for the author seeing how many proposals each has written, why does he need to see this. Just the total, then he can't be biased either.

**Comment by pietrosperoni** [comment]
regarding the table of who has voted, this info is out in any case once participants can see the triangle with the votes.
